x-pack/plugins/observability_solution/infra/public/components/asset_details/tabs/dashboards/dashboards.tsx

```diff
@@ -25,6 +25,13 @@ import {
 } from '@kbn/dashboard-plugin/public';
 
 import type { DashboardItem } from '@kbn/dashboard-plugin/common/content_management';
+import type { SerializableRecord } from '@kbn/utility-types';
+import {
+  ASSET_DETAILS_FLYOUT_LOCATOR_ID,
+  ASSET_DETAILS_LOCATOR_ID,
+} from '@kbn/observability-shared-plugin/public';
+import { useHostsTableUrlState } from '../../../../pages/metrics/hosts/hooks/use_hosts_table_url_state';
```

Comment:
Is there any other way to achieve this without creating a dependency between asset details and hosts view? Perhaps if you capture what's in the query string and forward that to the locator will do the job.

The asset details page/flyout should work independently from the hosts view.

Reply:
I agree with the dependency. 

Also, when opening a dashboard in a new tab the user has to wait for the whole hosts page to load before seeing the flyout view. Can we simplify it by redirecting the user to the asset details page?

Reply:
> Is there any other way to achieve this without creating a dependency between asset details and hosts view? Perhaps if you capture what's in the query string and forward that to the locator will do the job.

@crespocarlos Good point, thanks! I used the URL to get the table properties now. 👍 

> Also, when opening a dashboard in a new tab the user has to wait for the whole hosts page to load before seeing the flyout view. Can we simplify it by redirecting the user to the asset details page?

@kpatticha The open in a new tab functionality comes from the dashboard not from the hosts view. I don't think navigating to a different view makes sense here - if the user is already looking at the flyout I think it makes sense to stay on the same page (let's say the access to different hosts makes sense in this case and the possibility to switch between different hosts/hosts flyouts seeing the same tab for example). You are right that the whole page needs to load but that's the case for every link that needs to open in a new tab. The link can be always changed in the dashboard to not open on a new page. In case that's the use case I just included both in my testing to make sure we have the exact same result for both.